src/main/java/net/dzikoysk/funnyguilds/listener/region/EntityExplode.java

```diff
@@ -125,6 +133,10 @@ public void onExplode(EntityExplodeEvent event) {
                 }
             }
         }
+
+        if (!SimpleEventHandler.handle(new GuildEntityExplodeEvent(affectedBlocks))) {
+            event.setCancelled(true);
+        }
```

Comment:
This code doesn't make sense, because state of the affected blocks might have been changed by the code above, and cancelling this event wouldn't prevent from this.